public/js/debugResults.js

```diff
@@ -293,6 +297,63 @@
 	var alchemy = new Alchemy(config)
     }
 
+    function showTable(alchemy_data) {
+	TableDiv.innerHTML = ""
+
+	piv.html(TableDiv, "p", "Each would-be victory in a head-to-head matchup is represented by a \
+number in the following table.  Start with a candidate along the \
+vertical access.  The numbers in that row indicate the strengths of \
+the victories that the candidate would have in head-to-head matchups \
+against each candidate along the horizontal access.")
+	
+	var table = piv.html(TableDiv, "table", "")
+	// TODO: move this to a style sheet
```

Comment:
Good thinking! New GitHub issue?

Reply:
Good idea, added this: https://github.com/pivot-libre/pivot/issues/190

Reply:
👍